packages/AutoTDD-Core.package/ATDDGitHubActionsRepository.class/README.md

```diff
@@ -0,0 +1 @@
+An ATDDGitHubActionsRepository facilitates the GHAClient and GHAParser to provide structured data.
```

Comment:
We never use the short version `GHAParser`. We should write it out here as well. Same for `GHAClient`